src/Microsoft.Azure.EventHubs/ReceiverRuntimeInfo.cs

```diff
@@ -32,5 +36,17 @@ internal ReceiverRuntimeInformation(string partitionId)
         /// <summary>Gets the time of when the runtime info was retrieved.</summary>
         /// <value>The enqueued time of the last event.</value>
         public DateTime RetrievalTime { get; internal set; }
+
+        /// <summary>
+        /// Update the properties of this instance with the values from the given event.
+        /// </summary>
+        /// <param name="updateFrom"></param>
+        public void Update(EventData updateFrom)
```

Comment:
>public void Update(EventData updateFrom) [](start = 8, length = 40)

Can you use this in 2 other places where we assign properties one by one. In PartitonReceier.cs and PartitionPump.cs.

Reply:
Made requested change.